src/test/integration/IntegrationBase.t.sol

```diff
@@ -1133,6 +1133,45 @@ abstract contract IntegrationBase is IntegrationDeployer {
 
         return (strategies.sort(), wadsToSlash);
     }
+
+    function _strategiesAndWadsForFullSlash(
+        OperatorSet memory operatorSet
+    ) internal view returns (IStrategy[] memory strategies, uint[] memory wadsToSlash) {
+        // Get list of all strategies in an operator set.
+        strategies = allocationManager.getStrategiesInOperatorSet(operatorSet);
+
+        wadsToSlash = new uint[](strategies.length);
+
+        for (uint i; i < strategies.length; ++i) {
+            wadsToSlash[i] = 1 ether;
+        }
+
+        return(strategies.sort(), wadsToSlash);
+    }
+
+    function _strategiesAndWadsForRandFullSlash(
+        OperatorSet memory operatorSet
+    ) internal returns (IStrategy[] memory strategies, uint[] memory wadsToSlash) {
+        // Get list of all strategies in an operator set.
+        strategies = allocationManager.getStrategiesInOperatorSet(operatorSet);
+
+        // Randomly select a subset of strategies to slash.
+        uint len = _randUint({ min: 1, max: strategies.length-1 });
+
+        // Update length of strategies array.
+        assembly {
+            mstore(strategies, len)
+        }
+        
+        wadsToSlash = new uint[](len);
+        
+        // Randomly select a `wadToSlash` for each strategy.
+        for (uint i; i < len; ++i) {
+            wadsToSlash[i] = 1 ether;
+        }
```

Comment:
Is this supposed to be more random? If you want it to be random but you're leaving this work for a followup PR, can you add a TODO here?